src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/rmd/TextEditingTargetNotebook.java

```diff
@@ -753,13 +878,50 @@ else if (data.getType() == RmdChunkOutputFinishedEvent.TYPE_INTERACTIVE &&
             // set dirty state if necessary
             setDirtyState();
          }
+
+         if (satelliteChunks_.contains(data.getChunkId()))
+         {
+            forwardEventToSatelliteChunk(
+               docUpdateSentinel_.getId(),
+               data.getChunkId(),
+               new ChunkSatelliteOutputFinishedEvent(
+                  ensureVisible,
+                  data.getScope()
+               )
+            );
+         }
       }
    }
 
+   @Override
+   public void onConsoleWriteOutput(ConsoleWriteOutputEvent event)
```

Comment:
This shouldn't be necessary -- client events (defined as those emitted by `ClientEventDispatcher`) are already broadcast automatically to all satellites. If it doesn't work without manual re-broadcast let me know; it's possible that some other wiring is missing.